### ISSUE TITLE
`poetry add pharmpy && poetry install` installs documentation building tools

### ISSUE THREAD

**Issue by king-of-poppk** [pharmpy/pharmpy#3215]
### Bug report

Maybe I am doing something wrong, but it seems to me that `poetry` installs `sphinx`, `jupyter-sphinx`, among other things, that seem to belong to development dependencies. Is it intentional that these are installed when using `pharmpy` as a library?

**Comment by king-of-poppk** [comment]
`pharmpy-core` version is `0.105.3`.

**Comment by stellabelin** [comment]
Hi! `sphinx` is used to create reports in the different tools, so it's a "proper" dependency.

**Comment by king-of-poppk** [comment]
OK. So I guess this would require splitting `pharmpy` into multiple packages, if, say, one is not interested in tools.

**Comment by rikardn** [comment]
Yes, it would require splitting. And this is ideally what we should do. However we don't currently have the bandwidth to do a split and handle the increased maintenance complexity of having multiple packages. The modelbuilder package is now a proper separate package https://github.com/pharmpy/modelbuilder so this is giving us experience on how this can best be maintained.
